src/platform/amd/rembrandt/include/platform/lib/memory.h

```diff
@@ -172,8 +172,6 @@ static inline void *platform_rfree_prepare(void *ptr)
 
 #endif /* __PLATFORM_LIB_MEMORY_H__ */
 
-#endif /* __PLATFORM_LIB_MEMORY_H__ */
```

Comment:
right, please remove it a few patches before, where it was added by accident

Reply:
> right, please remove it a few patches before, where it was added by accident

Accidental code merge resolved.